hypothesisfuzzer/app.py

```diff
@@ -24,8 +30,19 @@ def get_commit_hash():
     sha = repo.head.object.hexsha
 
     return jsonify({
-            "sha": sha
-            })
+        "sha": sha
+    })
+
+
+"""
+SAMPLE CODE FOR A TABLE
+"""
+# class UserModel(db.Model):
+#     __tablename__ = 'users'
+
+#     uid = db.Column(db.Integer, primary_key=True)
+#     username = db.Column(db.String(80))
+#     password = db.Column(db.String(80))
```

Comment:
remove comments pls

Reply:
It is a sample use case of this database. I think it's good to keep it there.